### DISCUSSION TITLE
heavy augmentation 실험 해보려고 합니다!

### DISCUSSION THREAD

**Discussion by jiwoo0212** [boostcampaitech2/object-detection-level2-cv-06#111] [ideas]
mosaic augmenation과 어떤 성능 차이가 있을 지 궁금하여 heavy augementation을 진행하고 실험결과 공유하겠습니다.

model은 universe net을 사용할 예정입니다!

**Comment by kwakjeeyoon** [comment]
사용한 augmentation도 공유해주시면 좋을것 같습니다!

**Reply by kmouleejunhyuk** [reply]
일반적인 argumentation과 다른 점이 있나요?

**Reply by ppskj178** [reply]
저도 heavy augementation 이라고 따로 있는건가 싶어서 찾아봤는데 안나오네요...

혹시 어떤 방식인지 대략적으로 설명해주시면 좋을 것 같습니다.

**Reply by jaehwan-AI** [reply]
@kmouleejunhyuk 저는 오피스아워에서 설명 들었을 때 가능한 많은 augmentation을 적용한다는 의미라고 이해했었습니다.
근데 생각해보니 augmentation의 강도도 강하게 해야 heavy인 것인지는 의문이 드네요

**Reply by kmouleejunhyuk** [reply]
그냥 기존보다 arg를 많이 주는 걸로 이해해도 될까요?

**Reply by jiwoo0212** [reply]
저는 augmenatation을 여러가지 준다고 생각했습니다 강의 듣고!!!

Heavy augmentation
• RandomCrop, HorizontalFlip, VerticalFlip, ToGray, GaussNoise, MotionBlur, MedianBlur, Blur,
CLAHE, Sharpen, Emboss, RandomBrightnessContrast, HueSaturationValue

위의 리스트에서 추려서 진행해볼 예정입니다!

**Reply by kmouleejunhyuk** [reply]
@jiwoo0212  affine 도 있으면 고려해봐도 좋을 거 같습니다

**Reply by jiwoo0212** [reply]
@kmouleejunhyuk 좋은 의견 감사합니다!

**Reply by jaehwan-AI** [reply]
오 affine transform을 하면 bbox도 직사각형에서 다른 형태로 변형이 일어날 것 같은데 이게 어떤 영향을 미칠지 궁금하네요

**Reply by kmouleejunhyuk** [reply]
@jaehwan-AI affine을 한다고 bbox형태가 직사각형이 아니진 않을 거 같습니다.
![image](https://user-images.githubusercontent.com/49234207/136488851-449d7c7e-226a-4a63-a869-f7367b050325.png)

**Reply by jaehwan-AI** [reply]
아 물체 형태만 변하는 거군요.. 뭔가 bbox의 형태가 변해서 디텍터의 성능이 올라가는 것을 상상했었는데 아니었네요 ㅋㅋㅋ

**Reply by kmouleejunhyuk** [reply]
> 아 물체 형태만 변하는 거군요.. 뭔가 bbox의 형태가 변해서 디텍터의 성능이 올라가는 것을 상상했었는데 아니었네요 ㅋㅋㅋ

대신 찌그러진 형태에도 대응되어서 성능에 영향이 있을 거 같아요

**Comment by ppskj178** [comment]
universe net은 지윤님이 하신걸로 알고있는데
깃헙에 올려주신 세팅에 해당부분을 추가해서 진행하시겠다는 말씀이신가요?

그리고 CBNet은 진행 상황이 어떻게 되나요?
